android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,8 +1,8 @@
-• New OpenStreetMap data as of January 21
-• Dispay a speed limit sign in the navigation mode
-• Added Serbian (Cyrillic) and Latvian languages
-• Android Auto: sort bookmarks, improved location permission request experience
-• Added tower POIs
-• Added a setting for Kayak.com Hotel Links and a first-use opt-out option
+• New OpenStreetMap data as of February 27
+• Ability to manually arrange intermediate route points
+• Share a single selected track from the list
+• Improved routing for bicycles
```

Comment:
What does that mean in practice?

Reply:
less strange route cases, more optimal routes overall